core-bundle/src/Resources/contao/library/Contao/System.php

```diff
@@ -531,10 +532,18 @@ public static function loadLanguageFile($strName, $strLanguage=null, $blnNoCache
 		// Prepare the XLIFF loader
 		$xlfLoader = new XliffFileLoader(static::getContainer()->getParameter('kernel.project_dir'), true);
 		$strCacheDir = static::getContainer()->getParameter('kernel.cache_dir');
+		$availLangFilesPath = Path::join($strCacheDir, 'contao/config/available-language-files.php');
+		$availLangFiles = file_exists($availLangFilesPath) ? include $availLangFilesPath : null;
```

Comment:
`include` will load this file again on every call. Imho we can use `include_once` and cache the values statically?